types/v56/types.go

```diff
@@ -1439,16 +1440,37 @@ type OVFItem struct {
 	XmlnsVmw        string   `xml:"xmlns:vmw,attr,omitempty"`
 	VCloudHREF      string   `xml:"vcloud:href,attr"`
 	VCloudType      string   `xml:"vcloud:type,attr"`
-	AllocationUnits string   `xml:"rasd:AllocationUnits"`
-	Description     string   `xml:"rasd:Description"`
-	ElementName     string   `xml:"rasd:ElementName"`
-	InstanceID      int      `xml:"rasd:InstanceID"`
-	Reservation     int      `xml:"rasd:Reservation"`
-	ResourceType    int      `xml:"rasd:ResourceType"`
-	VirtualQuantity int      `xml:"rasd:VirtualQuantity"`
-	Weight          int      `xml:"rasd:Weight"`
-	CoresPerSocket  *int     `xml:"vmw:CoresPerSocket,omitempty"`
-	Link            *Link    `xml:"vcloud:Link"`
+	Address         string   `xml:"rasd:Address,omitempty"`
+	AddressOnParent *int     `xml:"rasd:AddressOnParent,omitempty"`
+
+	AllocationUnits string                       `xml:"rasd:AllocationUnits"`
+	Description     string                       `xml:"rasd:Description"`
+	ElementName     string                       `xml:"rasd:ElementName"`
+	HostResource    *VirtualHardwareHostResource `xml:"rasd:HostResource,omitempty"`
+	InstanceID      int                          `xml:"rasd:InstanceID"`
+	Parent          *int                         `xml:"rasd:Parent,omitempty"`
+	Reservation     int                          `xml:"rasd:Reservation"`
+	ResourceSubType string                       `xml:"rasd:ResourceSubType,omitempty"`
+	ResourceType    int                          `xml:"rasd:ResourceType"`
+
+	VirtualQuantity int   `xml:"rasd:VirtualQuantity"`
+	Weight          int   `xml:"rasd:Weight"`
+	CoresPerSocket  *int  `xml:"vmw:CoresPerSocket,omitempty"`
+	Link            *Link `xml:"vcloud:Link"`
+}
+
+// Used to write a list of RASD items, generally for modifying disks attached to a VM
```

Comment:
Please add the URL of the API docs page from where this structure was taken, and a note about what RASD stands for.